tests/test_regex_identifier.py

```diff
@@ -39,19 +39,54 @@ def test_lat_long2():
     assert "Latitude & Longitude Coordinates" in res[0]["Regex Pattern"]["Name"]
 
 
+def test_lat_long3():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(['77\u00B0 30\' 29.9988" N'])
```

Comment:
Whats `\u00B0`? Maybe a comment explaining?

Reply:
It's a degree symbol (°) as otherwise Windows tests fail. Will write a comment in test for that